catalogs/sources/gtfs/schedule/fr-hauts-de-france-pastel-gtfs-775.json

```
@@ -2,6 +2,11 @@
     "mdb_source_id": 775,
     "data_type": "gtfs",
     "provider": "PASTEL",
```

Should we deprecate this feed?
```suggestion
    "provider": "PASTEL",
    "status": "deprecated",
```